### PR TITLE
feat(node): dev node print genesis account

### PR DESCRIPTION
**Pull request by jsvisa**
similar to anvil, print genesis accounts when launch node with `--dev`:

```
2024-02-27T19:01:10.212909Z  INFO Allocated Genesis Account: 00. 0xdF3e18d64BC6A983f673Ab319CCaE4f1a57C7097 (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.212912Z  INFO Allocated Genesis Account: 01. 0x90F79bf6EB2c4f870365E785982E1f101E93b906 (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.212915Z  INFO Allocated Genesis Account: 02. 0xcd3B766CCDd6AE721141F452C550Ca635964ce71 (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.212917Z  INFO Allocated Genesis Account: 03. 0x8626f6940E2eb28930eFb4CeF49B2d1F2C9C1199 (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.212919Z  INFO Allocated Genesis Account: 04. 0xbDA5747bFD65F08deb54cb465eB87D40e51B197E (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.212921Z  INFO Allocated Genesis Account: 05. 0x70997970C51812dc3A010C7d01b50e0d17dc79C8 (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.212938Z  INFO Allocated Genesis Account: 06. 0x1CBd3b2770909D4e10f157cABC84C7264073C9Ec (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.212942Z  INFO Allocated Genesis Account: 07. 0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.212944Z  INFO Allocated Genesis Account: 08. 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.212946Z  INFO Allocated Genesis Account: 09. 0x71bE63f3384f5fb98995898A86B02Fb2426c5788 (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.212949Z  INFO Allocated Genesis Account: 10. 0x14dC79964da2C08b23698B3D3cc7Ca32193d9955 (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.212951Z  INFO Allocated Genesis Account: 11. 0xa0Ee7A142d267C1f36714E4a8F75612F20a79720 (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.212953Z  INFO Allocated Genesis Account: 12. 0xFABB0ac9d68B0B445fB7357272Ff202C5651694a (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.213015Z  INFO Allocated Genesis Account: 13. 0xBcd4042DE499D14e55001CcbB24a551F3b954096 (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.213027Z  INFO Allocated Genesis Account: 14. 0x2546BcD3c84621e976D8185a91A922aE77ECEc30 (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.213033Z  INFO Allocated Genesis Account: 15. 0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65 (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.213039Z  INFO Allocated Genesis Account: 16. 0xdD2FD4581271e230360230F9337D5c0430Bf44C0 (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.213045Z  INFO Allocated Genesis Account: 17. 0x976EA74026E726554dB657fA54763abd0C3a0aa9 (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.213051Z  INFO Allocated Genesis Account: 18. 0x23618e81E3f5cdF7f54C3d65f7FBc0aBf5B21E8f (1000000.000000000000000000 ETH)
2024-02-27T19:01:10.213056Z  INFO Allocated Genesis Account: 19. 0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc (1000000.000000000000000000 ETH)
```

### CODE REVIEW

File: crates/node-builder/src/builder.rs [hunk 1]
```diff
@@ -11,6 +11,7 @@ use crate::{
     rpc::{RethRpcServerHandles, RpcContext, RpcHooks},
     NodeHandle,
 };
+use alloy_primitives::utils::format_ether;
```
Comment (mattsse) [inline crates/node-builder/src/builder.rs:14]: I want to reexport this from reth-primitives instead

**Review by mattsse — CHANGES_REQUESTED** [review]
supportive, but I'd like to reexport this from reth-primitives instead so we don't need the alloy-primitives dep

File: crates/node-builder/src/builder.rs (hunk 1 above)
Comment (jsvisa) [reply crates/node-builder/src/builder.rs:14]: SGTM, I'll adjust it later today

**Review by mattsse — APPROVED** [review]
lgtm

**Pull request merged**